metaflow/plugins/argo/argo_workflows_deployer.py

```diff
@@ -188,6 +225,59 @@ def delete(instance: DeployedFlow, **kwargs):
     return command_obj.process.returncode == 0
 
 
+def from_deployment(identifier: str, metadata: str = None):
+    client = ArgoClient(namespace=KUBERNETES_NAMESPACE)
+    workflow_template = client.get_workflow_template(identifier)
```

Comment:
what happens if the workflow template doesn't exist?

Reply:
thanks, raising an exception here..